<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parametre</class>
 <widget class="QWidget" name="parametre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paramètres [Président - Client]</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Chat</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkJoueur">
        <property name="text">
         <string>Annoncer le joueur qui doit jouer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkCartes">
        <property name="text">
         <string>Annoncer les cartes reçus.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkCarteOuRien">
        <property name="text">
         <string>Annoncer lorsuq'il faut une carte ou rien.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkPass">
        <property name="text">
         <string>Annoncer lorsuq'un joueur passe.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkUltimeCarte">
        <property name="text">
         <string>Annoncer losqu'il ne vous reste qu'une ultime carte.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkReset">
        <property name="text">
         <string>Annoncer le reset du plateau.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkErreur">
        <property name="text">
         <string>Rapport d'erreur</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Volume des sons</string>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="whatsThis">
         <string>Volume du son joué lors du démarrage d'une partie</string>
        </property>
        <property name="text">
         <string>Démarrage de la partie</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="volumeStart">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>20</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="whatsThis">
         <string>Volume du son joué lors qu'un joueur a fait une action</string>
        </property>
        <property name="text">
         <string>Joueur suivant</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="volumeNext">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>20</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="whatsThis">
         <string>Volume du son lorsqu'un joueur a gagné</string>
        </property>
        <property name="text">
         <string>Gagnant</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="volumeWin">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>20</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="whatsThis">
         <string>Volume du son lors d'un nouveau message</string>
        </property>
        <property name="text">
         <string>Nouveau message</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="volumeChat">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>20</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="whatsThis">
         <string>Volume du son joué si vous ne jouez pas après un interval de temps</string>
        </property>
        <property name="text">
         <string>Volume alerte</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSlider" name="volumeAlerte">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>20</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Autres</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="whatsThis">
         <string>Interval entre les aletres si vous ne jouez pas</string>
        </property>
        <property name="text">
         <string>Intervalen les  alerte (s)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="tempAlerte">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
